ansible/roles/mailcow/molecule/default/molecule.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+driver:
+  name: vagrant
+  provider:
+    name: libvirt
+
+lint: |
+  set -e
+  yamllint .
+
+platforms:
+  - name: debian9-instance
+    box: generic/debian10
+    memory: 2000
+    cpus: 1
+    groups:
+      - mailserver
+  - name: debian10-instance
+    box: generic/debian10
+    memory: 2000
+    cpus: 1
+    groups:
+      - mailserver
```

Comment:
Why are we doing deploy on 2?

Reply:
This is an error, 2 different version of Debian should be tested.